docs/use_cases/model_applications/s2s/README.rst

```diff
@@ -1,2 +1,3 @@
 Subseasonal to Seasonal
 -----------------------
+Subseasonal-to-Seasonal model configurations.  Lower resolution model configuration (>4km) usually producing forecasts out beyond 14 days and up 1 year.
```

Comment:
Subseasonal-to-Seasonal model configurations.  Lower resolution model configurations (>4km) usually producing forecasts out beyond 14 days and up 1 year.